serenata_toolbox/cleanup.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+# Utilities for extracting data from XML files
+
+def xml_extract_text(node, xpath):
+    """
+    :param node: the node to be queried
+    :param xpath: the path to fetch the child node that has the wanted text
+    """
+    text = node.find(xpath).text
+    if text != None:
+        text = text.strip()
+    return text
+
+def xml_extract_date(node, xpath):
```

Comment:
What about that:

```python
def xml_extract_date(node, xpath, date_format='%d/%m/%Y'):
    …
```
Then this function can be used in a more generic way if needed ; )